webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/pdf/GeneralScrambleSheet.kt

```diff
@@ -128,22 +110,14 @@ class GeneralScrambleSheet(scrambleRequest: ScrambleRequest, globalTitle: String
         return Font(FontUtil.MONO_FONT, perfectFontSize, Font.NORMAL) to oneLine
     }
 
-    fun requiresHighlighting(availableWidth: Float, scrambleImageSize: Dimension, scrambles: List<String>, scrambleNumberPrefix: String): Boolean {
-        val scrambleColumnWidth = getScrambleColumnWidth(availableWidth, scrambleImageSize, scrambles, scrambleNumberPrefix)
-        val (scrambleFont, oneLine) = getFontConfiguration(availableWidth, scrambleImageSize, scrambles, scrambleNumberPrefix)
-
+    fun requiresHighlighting(scrambleColumnWidth: Float, scrambles: List<String>, scrambleFont: Font, oneLine: Boolean): Boolean {
         val paddedScrambles = scrambles.map { if (oneLine) it else StringUtil.padTurnsUniformly(it, PdfUtil.NON_BREAKING_SPACE.toString()) }
         val lineChunks = paddedScrambles.map { it.splitToLineChunks(scrambleFont, scrambleColumnWidth) }
 
         return lineChunks.any { it.size >= MIN_LINES_TO_ALTERNATE_HIGHLIGHTING }
```

Comment:
There _may_ be a bug here, since the example PDF you uploaded shows 3 lines without highlighting